core/api/src/app/merchants/delete-business-map-info.ts

```diff
@@ -0,0 +1,38 @@
+import { checkedToUsername } from "@/domain/accounts"
+import { MerchantsRepository } from "@/services/mongoose"
+
+export const deleteBusinessMapInfo = async ({
```

Comment:
Small nit, consistent naming

```suggestion
export const deleteMerchantLocationById = async ({
```